tailwind.config.js

```diff
@@ -131,6 +131,9 @@ module.exports = {
       },
       transitionTimingFunction: {
         'animation-curve': 'cubic-bezier(0.68, -0.55, 0.265, 1.55)'
+      },
+      opacity: {
+        '30': 0.3
```

Comment:
💚 I think this overrides all the preset opacity styles, worth checking we aren't using them anywhere currently, as this might break them.

Reply:
Double checked and its all good! I think it's fine because its in the extend object, if its just in theme then it overrides